src/commands/social/profile.js

```diff
@@ -0,0 +1,23 @@
+const { CanvasTemplates, CommandStructures } = require('../../')
+const { Command, CommandParameters, UserParameter } = CommandStructures
+const { Attachment } = require('discord.js')
+
+module.exports = class Profile extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'profile'
+    this.aliases = []
+
+    this.parameters = new CommandParameters(this,
+      new UserParameter({full: true, required: false})
+    )
+  }
+
+  async run ({ t, author, channel, client }, user) {
+    user = user || author
+    channel.startTyping()
+    const before = Date.now()
+    const profile = await CanvasTemplates.profile({ t, client }, user)
+    channel.send(`Took **${Date.now() - before}**ms`, new Attachment(profile, 'profile.jpg')).then(() => channel.stopTyping())
```

Comment:
move the strings to `strings.json`

Reply:
The only thing that needs to be moved is "Next reward in". "Switchcoins" is already a string in `commons.json`.

Reply:
not that dummy, i'm talking about "Took **{{time}}**ms"

Reply:
That's temporary